package.json

```diff
@@ -9,8 +9,16 @@
     "start": "next start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "lint": "eslint . --ext=tsx,ts",
-    "test": "yarn lint"
+    "lint": "jest --config tests/configs/jest.lint.js",
```

Comment:
Could you explain the switch from `eslint` to `jest-eslint-runner` for linting?

Reply:
It's still using eslint, it's just running it in the context of jest, the jest output looks a bit nicer than the eslint output, but if you guys see a problem with outputting the lint errors through jest then we can revert back to using eslint.
we can add `"lint:fix": "eslint . --ext=tsx,ts --fix"` as you mention before for fixing the linting errors and I can add `'jest-runner-eslint/watch-fix',` as a watch plugin for jest so we can fix linting errors if we are running in watch mode

Reply:
Ah, cool thanks for explaining! I have a slight preference to leave it as eslint since this adds another config/level of complexity and if we have other `lint:*` scripts that use eslint it would be great to have everything be consistent. If we do want to swap over, could we do it in a separate PR? That way we can have this one focus on test setup and subsequent PRs focus on other build/test/lint aspects.